doc/rtd/explanation/analyze.rst

```diff
@@ -3,15 +3,18 @@
 Performance
 ***********
 
-The :command:`analyze` subcommand was added to ``cloud-init`` to help analyze
-``cloud-init`` boot time performance. It is loosely based on
-``systemd-analyze``, where there are four subcommands:
+The :command:`analyze` subcommand helps to analyze ``cloud-init`` boot time
+performance. It is loosely based on ``systemd-analyze``, where there are four
+subcommands:
 
 - :command:`blame`
 - :command:`show`
 - :command:`dump`
 - :command:`boot`
 
+The analyze subcommand works by parsing the cloud-init log file for timestamps
```

Comment:
This subcommand fails to capture interpreter startup time and library load time because of this fact, so at least advertise how it works so that users can draw their own conclusions.